amf-shapes/shared/src/main/scala/amf/shapes/client/platform/model/domain/jsonldinstance/JsonLDArray.scala

```diff
@@ -0,0 +1,19 @@
+package amf.shapes.client.platform.model.domain.jsonldinstance
+
+import amf.core.client.scala.model.domain.AmfArray
+import amf.shapes.client.scala.model.domain.jsonldinstance.{JsonLDArray => InternalJsonLDArray}
+import amf.shapes.internal.convert.ShapeClientConverters.ClientList
+import scala.scalajs.js.annotation.{JSExportAll, JSExportTopLevel}
+
+@JSExportAll
+class JsonLDArray extends AmfArray(Nil) with JsonLDElement {
+@JSExportTopLevel("JsonLDArray")
+  def this() = this(InternalJsonLDArray())
+
+  def +=(value: JsonLDElement): Unit ={
```

Comment:
When having the _internal you should change this to call the _internal values (call the += method of _internal)